docs/en/contribution/setup.md

```diff
@@ -46,6 +46,12 @@ See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plug
 Before running the following example, you should also install JetBrains IntelliJ IDEA's [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok).
 See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plugins.html#install-plugins) if you want to.
 
+### Install JetBrains IDEA CheckStyle-IDEA Plugin
+
+Before coding, you should also install JetBrains IntelliJ IDEA's [CheckStyle-IDEA plugin](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea).
+See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plugins.html#install-plugins) if you want to.
+Next, you should go to `Preferences -> Editor -> Code style -> Scheme -> Import Scheme -> CheckStyle Configration` and import `tools/checkstyle/checkStyle.xml`
```

Comment:
Preferences -> Editor -> Code Style -> Java
If the user wants to change to automatic formatting, these configurations are also required

Reply:
Do you mean I should add `If the user wants to change to automatic formatting, these configurations are also required`  to the doc?

Reply:
yes